roles/image_build/templates/ansible.cfg.j2

```diff
@@ -0,0 +1,16 @@
+[defaults]
+any_errors_fatal = True
+gathering = smart
+forks = 30
```

Comment:
Only running 1x builder, could omit to get default 5 forks.

Reply:
Spot - fixed